### PR TITLE
Adding WORKFLOW_EXECUTION_AUX table and enabling test assertions.

### PR DESCRIPTION
**Pull request by mcovarr**
This mostly mimics the work Khalid has done, hopefully I haven't inadvertently committed any crimes against Slick.

### CODE REVIEW

File: src/main/migrations/changesets/wdl_and_inputs.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    <changeSet author="mcovarr" id="WORKFLOW_EXECUTION_AUX">
+        <comment>
+            Auxiliary data related to a workflow execution not stored with the workflow execution itself
+            for size reasons.
+        </comment>
+        <createTable tableName="WORKFLOW_EXECUTION_AUX">
+            <column autoIncrement="true" name="WORKFLOW_EXECUTION_AUX_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_WE_AUX"/>
+            </column>            <column name="WORKFLOW_EXECUTION_ID" type="INT"/>
```
Comment (mcovarr) [inline src/main/migrations/changesets/wdl_and_inputs.xml]: fixing this

File: src/main/migrations/changesets/wdl_and_inputs.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <property name="clob.type" value="LONGTEXT" dbms="mysql"/>
```
Comment (mcovarr) [inline src/main/migrations/changesets/wdl_and_inputs.xml]: `TEXT`is 64K in MySQL, is that sufficient?

Comment (geoffjentry) [inline src/main/migrations/changesets/wdl_and_inputs.xml]: @scottfrazer and I discussed this a while back and I forget what we decided. Certainly that `LONGTEXT` would never be exceeded :) IIRC the discussion was if the smaller one would 100% be safe or if we should use the medium sized one. 

AFAICT there's no real danger to upgrading if it turns out 64K isn't enough for everybody, to paraphrase a famous underestimator of memory usage.

Comment (scottfrazer) [inline src/main/migrations/changesets/wdl_and_inputs.xml]: As some perspective, this WDL file is 1.9K:

https://github.com/broadinstitute/cromwell/blob/wdl_features/new.wdl

Comment (geoffjentry) [inline src/main/migrations/changesets/wdl_and_inputs.xml]: One thing I'm curious about is what would happen if you tried to put an oversized thing in there. Would it barf or would it silently truncate? I'd imagine the former but the latter would be disastrous.

**Pull request merged**